app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
```

Comment:
The BaseException is the base class of all other exceptions. You need to use Exception class to be directly derived.